test/Microsoft.ML.TestFramework/Attributes/OnnxTheoryAttribute.cs

```diff
@@ -19,7 +19,7 @@ public OnnxTheoryAttribute() : base("Onnx is 64-bit Windows only")
         /// <inheritdoc />
         protected override bool IsEnvironmentSupported()
         {
-            return Environment.Is64BitProcess && RuntimeInformation.IsOSPlatform(OSPlatform.Windows);
+            return Environment.Is64BitProcess && (RuntimeEnvironment.OperatingSystem.Contains("Windows") || RuntimeEnvironment.OperatingSystem.Contains("ubuntu"));
```

Comment:
> (RuntimeEnvironment.OperatingSystem.Contains("Windows")

This should stay as `RuntimeInformation.IsOSPlatform(OSPlatform.Windows)`.

> RuntimeEnvironment.OperatingSystem.Contains("ubuntu")

I believe the more correct check here would be "get the glibc version. If it is greater than or equal to the supported `glibc` version Onnx, return `true`.

Reply:
The "Windows" versus "Ubuntu" would probably work for ML.NET CI (which is where the tests would run). The glibc version check of 2.23+ is a better (more general check) for OnnxRuntime. However, there doesn't seem to be a very good way to check for this.  We could test the loader version (e.g. "ldd --version"), but that would spawn an external process. Also, I don't believe that will work if the OS has been patched.

Reply:
This is how we get the `libc` version in the CLI for telemetry purposes:

https://github.com/dotnet/cli/blob/95c6eff6daa1a69f29c42b2d405400ad44bdec91/src/dotnet/Telemetry/ExternalTelemetryProperties.cs#L110-L132